index.html

```diff
@@ -0,0 +1,132 @@
+          <ul class="main-menu dropdown">
+            <li class="main-menu-item"><a href="#">lifestyle</a></li>
+            <li class="main-menu-item"><a href="#">photodiary</a></li>
+            <li class="main-menu-item"><a href="#">music</a></li>
+            <li class="main-menu-item"><a href="#">travel</a></li>
+          </ul>
+         </nav>
+    </header>
+    <main class="main-content-wrapper">
+      <section id="main-article">
+        <div class="wrapper">
+          <img src="img/pics/main_post_img.jpg" alt="Main Article Image" class="main-image"/>
+          <article class="main-article">
+            <span class="category">photodiary</span>
+            <h2 class="main-article-title">The Perfect weekend getaway</h2>
+            <p class="main-article-paragraph">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed vel nisl nibh. Sed eget molestie mauris. Fusce nec dolor posuere sem laoreet gravida eget quis turpis. Donec congue lorem at lectus euismod tempor. Vivamus sem enim, tincidunt vitae scelerisque id, posuere nec erat. Maecenas imperdiet mattis lectus at egestas. Aliquam malesuada, felis nec tempor tristique, velit nisl vehicula felis, ut porttitor elit nibh quis ligula. In hac habitasse platea dictumst. Nunc eget erat non quam molestie volutpat. Curabitur fermentum porttitor turpis rhoncus mattis. Nam rhoncus mi sit amet ligula tempus vulputate. Nulla facilisi. Proin rhoncus elementum nibh, nec euismod purus interdum eget.
+            </p>
+            <p class="main-article-paragraph">
+            Sed sed est a ante fringilla imperdiet commodo eget ex. Sed pharetra justo sit amet massa tincidunt egestas. Proin suscipit massa vitae maximus dignissim. Aenean sagittis ligula in consequat tincidunt. Vestibulum hendrerit sapien sit amet magna facilisis fringilla. Etiam blandit vehicula massa quis blandit. Fusce maximus, leo nec vehicula eleifend, ipsum magna commodo enim, non molestie tortor orci eget dolor. Nam ut nunc ac nunc pharetra tempor. Phasellus consequat, lacus eget tempus vulputate, lorem ligula sagittis enim, eget ullamcorper diam nisi eget orci. Morbi non arcu augue.
+          </p>
+            <span class="leave_comment"><a href="#">leave a comment</a></span>
+          </article>
+        </div>
+      </section>
+
+      <section id="other-articles">
+        <div class="wrapper other-articles-container">
+          <article class="other-article">
+            <img src="img/pics/more_than.jpg" alt="festival image" class="other-article-img" />
+            <span class="category">lifestyle</span>
+            <h2 class="other-article-title">More than just a music festival</h2>
```

Comment:
A gdzie masz `h1`? :) Tak na moje oko to napis `PHOTODIARY` powinien być `h1`. Nie chodzi o wielkość fontu, bo możesz ustawić mu mniejszych niż tych `h1`, chodzi o semantykę